karma.config.js

```diff
@@ -0,0 +1,70 @@
+const commonjs = require('rollup-plugin-commonjs');
+const resolve = require('rollup-plugin-node-resolve');
+const builds = require('./rollup.config');
+
+module.exports = function(karma) {
+	const args = karma.args || {};
+	const regex = args.watch ? /\.js$/ : /\.min\.js$/;
+	const build = builds.filter((v) => v.output.file.match(regex))[0];
+
+	if (args.watch) {
+		build.output.sourcemap = 'inline';
+	}
+
+	karma.set({
+		browsers: ['firefox'],
```

Comment:
If using only one browser, should it rather be chrome these days?
Should not make any difference though.

Reply:
I think I prefer Firefox because it's easier to setup with Travis.